test/commands/alias/list.nut.ts

```diff
@@ -43,20 +55,24 @@ describe('alias:list NUTs', () => {
         status: 0,
       });
     });
+
+    it('lists singular result correctly stdout', () => {
+      const res: string = execCmd('alias:list', { ensureExitCode: 0 }).shellOutput;
+      expect(res).to.include('=== Alias List');
+      expect(res).to.include('DevHub');
+      expect(res).to.include('mydevhuborg@salesforce.com');
+    });
   });
 
   describe('alias:list with multiple results', () => {
-    before(() => {
-      testSession = TestSession.create({
-        setupCommands: [
-          'sfdx alias:set DevHub=mydevhuborg@salesforce.com',
-          'sfdx alias:set Admin=admin@salesforce.com',
-          'sfdx alias:set user=user@salesforce.com',
-        ],
-      });
+    beforeEach(() => {
+      unsetAll();
+      execCmd('alias:set DevHub=mydevhuborg@salesforce.com');
+      execCmd('alias:set Admin=admin@salesforce.com');
+      execCmd('alias:set user=user@salesforce.com');
```

Comment:
jsyk, you should be able to set and unset multiple aliases in 1 command

Reply:
But is sacrificing the readability worth it?